### ISSUE TITLE
Design System accessibility audit

### ISSUE THREAD

**Issue by HelenOsg** [dwp/design-system#170]
## What
Design System accessibility audit

## Why
It's important for us to be a service that's a platform to embed a11y best practice in the things we do and make. It makes things better for everyone.

## Done when

- [x] Contact Craig to see if he can support on accessibility audit of design site
- [x] Analyse results ands findings 
- [x] Create issues for fixes
- [x] Fix issues
- [x] Notify Accessibility team for fixes to be checked
- [x] [Publish accessibility statement and fixes](#349)
- [ ] Publish a11y sections for each component
- [ ] Strategy for testing going forward

## Outcomes

A WCAG compliant site. No active barriers for people using our service and the things made by it.

## Who needs to know about this
UCD Community
Accessibility Team
@abbott567
 #design-system channel
 #accessibilty channel

## Related stories

- [x] [Create issues to address from the accessibility report](https://github.com/dwp/design-system/issues/269)
    - [x] [Accessibility audit: /index URLs to match Heading Level 1s](https://github.com/dwp/design-system/issues/276)
    - [x] [Accessibility audit: Broken tab focus](https://github.com/dwp/design-system/issues/282)
    - [x] [Accessibility audit: (Dragon #19) Hidden text identifies with example link references](https://github.com/dwp/design-system/issues/290)
    - [x] [Accessibility audit: (Dragon) Say command differs on results between HTML/Nunjucks tab](https://github.com/dwp/design-system/issues/291)
    - [x] [Accessibility audit: /consent-to-cookies Image link focus style](https://github.com/dwp/design-system/issues/292)
    - [x] [Accessibility audit: /consent-to-cookies broken links](https://github.com/dwp/design-system/issues/293)
    - [x] [Accessibility audit: (JAWS) Announce example content within links hidden text](https://github.com/dwp/design-system/issues/294)
    - [x] [Accessibility audit: (JAWs) Nunjucks Tab missing from list item list](https://github.com/dwp/design-system/issues/299)
    - [x] [Accessibility audit: (JAWS) Tab component missing expanded states](https://github.com/dwp/design-system/issues/300)
    - [x] [Accessibility audit: (JAWS) Tab component, tab differentiation with hidden text when using multiple examples](https://github.com/dwp/design-system/issues/301)
    - [x] [Accessibility audit: (JAWS) Tabs appear as links but missing for link items list](https://github.com/dwp/design-system/issues/302)
    - [x] [Accessibility audit: Tabs UI different from GDS Tab component](https://github.com/dwp/design-system/issues/303)
    - [x] [Accessibility audit: Sub navigation section hidden text heading missing](https://github.com/dwp/design-system/issues/305)
    - [x] [Accessibility audit: /patterns Sub navigation section heading level](https://github.com/dwp/design-system/issues/306)
    - [x] [Accessibility audit: (JAWS) /consent-to-cookies Flow diagram no description](https://github.com/dwp/design-system/issues/307)
    - [x] [Accessibility audit: (JAWS) /backlog Heading level order](https://github.com/dwp/design-system/issues/310)
    - [x] [Accessibility audit: (JAWS) /backlog Tables have no name](https://github.com/dwp/design-system/issues/311)
    - [x] [Accessibility audit: (JAWS) Get in touch channel # announcement](https://github.com/dwp/design-system/issues/312)
    - [x] [Accessibility audit: (JAWS) Slack channel link not public](https://github.com/dwp/design-system/issues/313)
    - [x] [Accessibility audit: Missing Privacy statement and link](https://github.com/dwp/design-system/issues/314)

**Comment by jonhurrell** [comment]
We've had an email from the a11y team regarding a lack of evidence on the SharePoint folder. I wasn't aware of this process and thought the evidence of fixes embedded within the report and sent back to the team would be the most common sense approach.
As this team is evolving its processes, I feel we are stuck for a path forward in terms of finishing up and an acknowledgement of the fixes above.

**Comment by jonhurrell** [comment]
Met with a11y team. Agreed to review the changes above and feedback. We'll then complete the automated testing tools template and the MacBook specific testing tools templates and email the evidence to Nicki. We should attach all testing templates to revenant documentation for reference.
